docs(mongodb): document Windows limitation for Percona mongodb_exporter (#25041)
newrelic/docs-website: src/install/mongodb/default-requirements.mdx

* Our integration is compatible with MongoDB 4.0 or higher, Percona Server, and MongoDB Atlas M10 or higher.
* To monitor MongoDB Atlas, you'll need a hosted environment to install the infrastructure agent and integration and remotely connect to the Atlas instance.

<Callout variant="important">

>   Windows support for the MongoDB integration is currently frozen at its existing bundled exporter version. The upstream [Percona `mongodb_exporter`](https://github.com/percona/mongodb_exporter) project stopped publishing Windows binaries starting with its `v0.47.1` release, so the integration won't receive exporter updates, including security fixes, on Windows until Windows builds are available again upstream.

Reviewer: ```suggestion
  Windows support for the MongoDB integration is currently frozen at its existing bundled exporter version. The upstream [Percona `mongodb_exporter`](https://github.com/percona/mongodb_exporter) project stopped publishing Windows binaries starting with the `v0.47.1` release. As a result, the integration won't receive exporter updates, including security fixes, until upstream builds resume.
```
